### ISSUE TITLE
Add accessibility options for users with poor eyesight

### ISSUE THREAD

**Issue by cooljeanius** [wesnoth/wesnoth#6133]
Chess & I were brainstorming some of these ideas on Discord:
- [ ] color-blind mode
- [ ] high contrast mode
- [ ] add option to add outlines to all sprites, possibly in their team colors
- [ ] add option to outline the edges of tiles that units are standing on, possibly also in their team colors
- [ ] option to disable tile transitions

Let me know if any of these should be split off into separate bugs.

**Comment by Zireael07** [comment]
3 & 4 would be awesome (I use glasses every day)

**Comment by max-torch** [comment]
I would add
- [ ] Set brightness/contrast of highlighted hexes (the ones that show where a unit can move). 

It is especially hard to distinguish when you are in a cave with flat, road, cave, hill, terrain all mixed together.
Case in point:
![image](https://user-images.githubusercontent.com/41433185/135382324-9aee9ebf-cf99-49ff-82db-d2b61d5baf6c.png)
Can you easily distinguish the highlighted and unhighlighted hexes?

**Comment by soliton-** [comment]
I would suggest to split the first two into separate issues and elaborate on what exactly those modes should entail. The rest seems fairly straight forward.

**Comment by cooljeanius** [comment]
> I would suggest to split the first two into separate issues and elaborate on what exactly those modes should entail. The rest seems fairly straight forward.

Hm, I tried clicking the "Convert to issue" circle/button that appears when you hover over a task list item, but it doesn't seem to be working currently:
<img width="443" alt="Screen Shot 2021-10-03 at 11 34 38" src="https://user-images.githubusercontent.com/1388660/135761097-597862af-e58a-489c-b0f0-27e059445c4e.png">

**Comment by CelticMinstrel** [comment]
It worked for me (perhaps it requires access to the repository), so go ahead and elaborate on the details.

**Comment by cooljeanius** [comment]
Screenshots of the Discord convo that prompted this:
<img width="896" alt="Screen Shot 2021-10-03 at 12 39 49" src="https://user-images.githubusercontent.com/1388660/135763352-1ceab0c7-a000-4856-8d0f-e41559e82f3c.png">
<img width="893" alt="Screen Shot 2021-10-03 at 12 40 12" src="https://user-images.githubusercontent.com/1388660/135763360-1da9cad3-9824-4a89-a809-a94dbb45738a.png">

**Comment by cooljeanius** [comment]
This forum thread seems to fit here: https://forums.wesnoth.org/viewtopic.php?t=56404

**Comment by cooljeanius** [comment]
> I support high contrast mode. Even for someone who doesn't have a visual impairment. Would be great if it has a toggleable hotkey.
> 
> It is especially hard to distinguish when you are in a cave with flat, road, cave, hill, terrain all mixed together. Case in point: ![image](https://user-images.githubusercontent.com/41433185/135382324-9aee9ebf-cf99-49ff-82db-d2b61d5baf6c.png) Can you easily distinguish the highlighted and unhighlighted hexes?

Yeah putting an outline/border around the edge of the reachable range would be helpful here IMO...
Also, another idea while I'm at it... has anyone ever tried playing Wesnoth with a screen reader before? I think I've heard that people are able to play chess like that by having their screen reader read the name of the piece and the tile being moved to, so maybe Wesnoth could allow for something similar? Although it might be a bit more complicated here since it's more than just tile number and piece that matter; there's also terrain, multiple attacks, experience, etc.

**Comment by cooljeanius** [comment]
Some user Kaiji posted these screenshots to Discord, which I think fit here; apparently they have created a better moves display, and some clearer selected unit highlighting:
![image1](https://user-images.githubusercontent.com/1388660/217548821-a2d84ff4-6179-4874-b509-84fe972c3032.png)
![image2](https://user-images.githubusercontent.com/1388660/217548862-280181cb-3fc8-4ee6-8b8c-ffc5a97dc7fa.png)
![image3](https://user-images.githubusercontent.com/1388660/217548958-91ab2152-9f23-4e38-9437-ac9477b61217.png)

**Comment by cooljeanius** [comment]
Issue #9027 would help with this a bit.

**Comment by cooljeanius** [comment]
#9542 might kind of address this, at least partially

**Comment by cooljeanius** [comment]
> [#9542](https://github.com/wesnoth/wesnoth/pull/9542) might kind of address this, at least partially

@ForestDragon-wesnoth's PR #9877 should help, too
